docs/ref/forms/renderers.txt

```diff
@@ -0,0 +1,161 @@
+======================
+The form rendering API
+======================
+
+.. versionadded:: 1.11
+
+Django form widgets are rendered using the Django template backend system.
+
+The form rendering process can be customized at several levels:
+
+* Widgets can specify custom template names.
+* Forms and widgets can specify custom renderer classes.
+* Built-in templates can be overridden within a project or application.
+
+.. _low-level-widget-render-api:
+
+The low-level render API
+========================
+
+Widget templates are stored in the ``django/forms/widgets`` path. A project
```

Comment:
Yeah, I agree that third-party apps in most cases shouldn't be overriding form widgets (though I suppose I could see some exceptions for e.g. something like a django-bootstrap-forms app, if it clearly advertises that this is what it does).

Maybe it'd be possible to rephrase this sentence in a way that doesn't have to mention either "project" or "app", since the real point here is just that form widgets can be overridden. Maybe passive voice? Or do we use second person in the docs? "You can provide a custom template for..."